connectors/tailoringDataProvider.js

```diff
@@ -0,0 +1,45 @@
+);
+
+allDBStrings.push(
+
+);
+
+function parseAllDBStrings() {
```

Comment:
At this point we should start thinking about a general class that we extend for each profession to prevent code-duplication

Reply:
Fixed, moved it to utils for now